ivy/stateful/activations.py

```diff
@@ -550,3 +550,25 @@ def _forward(self, x):
             The outputs following the LogSigmoid activation *[batch_shape, d]*
         """
         return ivy.logsigmoid(x, complex_mode=self._complex_mode)
+
+
+class SegmentMax(Module):
```

Comment:
stateful is not needed for maths functions

Reply:
removed that function